priv/gettext/fr/LC_MESSAGES/default.po

```diff
@@ -295,7 +295,7 @@ msgstr "Mise à jour en cours"
 #, elixir-format
 #: lib/teslamate_web/templates/car/summary.html.leex:52
 msgid "Windows open"
-msgstr "Vitre ouverte"
+msgstr "Vitres ouvertes"
```

Comment:
```suggestion
msgstr "Fenêtres ouvertes"
```
It might be a personal preference thing, but I feel like I more often hear "Fenêtre" when talking about a car window than "Vitre". Might be different in France.

Reply:
In France we used to say "Vitres de voiture", "Vitre ouverte", "Vitre teintées" ...